utils.py

```diff
@@ -171,8 +172,6 @@ def is_target(target, op):
         real = real.split(".")[-1]
     if re.match(target+'[0-9]*$', real):
         return True
-    if re.match(target+'[23]d[0-9]*$', real):
-        return True
```

Comment:
Please revert this change. It fixed the bug that failed to identify patterns with the suffix "2d/3d".

Reply:
ok